pkg/cmd/kafka/create/create.go

```diff
@@ -653,26 +684,18 @@ func promptKafkaPayload(opts *options, constants *remote.DynamicServiceConstants
 			return nil, err
 		}
 
-		sizes, err := FetchValidKafkaSizes(opts.f, answers.CloudProvider, answers.Region, *userQuota)
+		sizes, err = FetchValidKafkaSizes(opts.f, answers.CloudProvider, answers.Region, *userQuota)
 		if err != nil {
 			return nil, err
 		}
 
-		if len(sizes) == 1 {
-			answers.Size = sizes[0].GetId()
-		} else {
-			sizeLabels := GetValidKafkaSizesLabels(sizes)
-			planPrompt := &survey.Select{
-				Message: f.Localizer.MustLocalize("kafka.create.input.plan.message"),
-				Options: sizeLabels,
-			}
-
-			err = survey.AskOne(planPrompt, &answers.Size)
-			if err != nil {
-				return nil, err
-			}
+		index, err := promptUserForSizes(opts.f, &sizes)
```

Comment:
Should we skip the prompt if it has been set through the options?

Reply:
From how other options work, if any option is set we skip the interactive selection and make the payload from that meaning because we are currently in interactive mode we know size has not been set

Reply:
👍  fair!